src/fairchem/core/common/profiler_utils.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+import os
+from typing import TYPE_CHECKING
+
+import torch
+
+from fairchem.core.common import distutils
+
+if TYPE_CHECKING:
+    from fairchem.core.common.logger import Logger
+
+def get_default_profiler_handler(run_id: str, output_dir: str, logger: Logger):
+    """Get a standard callback handle for the pytorch profiler"""
+
+    def trace_handler(p):
+        if distutils.is_master():
+            trace_name = f"{run_id}_rank_{distutils.get_rank()}.pt.trace.json"
+            output_path = os.path.join(output_dir, trace_name)
+            print(f"Saving trace in {output_path}")
+            p.export_chrome_trace(output_path)
+            if logger:
+                logger.log_artifact(name=trace_name, type="profile", file_location=output_path)
+    return trace_handler
+
+def get_profile_schedule(wait: int = 5, warmup: int = 5, active: int = 2):
```

Comment:
might be nice to add a comment describing what int, warmup, and active are.

Reply:
done!